resources/views/layouts/app.blade.php

```diff
@@ -12,9 +12,14 @@
 
     <!-- Styles -->
     <link href="{{ asset('css/app.css') }}" rel="stylesheet">
+    <link href="{{ asset('css/sidebar.css') }}" rel="stylesheet" />
+
 
-    <link rel="stylesheet" href="{{ asset('css/bootstrap/bootstrap-theme.min.css')}}">
```

Comment:
@brynbeaudry I'm not too sure about this, but I don't think we should be removing bootstrap from the default layout file.